### PR TITLE
applications: nrf_desktop: Update bootloader docs

### PR DESCRIPTION
**Pull request by MarekPieta**
Change updates documentation related to bootloaders.

Jira:DESK-1059

### CODE REVIEW

File: applications/nrf_desktop/README.rst [hunk 1]
```diff
@@ -1311,7 +1311,7 @@ The HID over GATT profile specification requires Bluetooth Peripherals to define
 * HID Service - Handled in the :ref:`nrf_desktop_hids`.
 * Battery Service - Handled in the :ref:`nrf_desktop_bas`.
 * Device Information Service - Implemented in Zephyr and enabled with :option:`CONFIG_BT_GATT_DIS`.
-  It can be configured using Kconfig options with the ``CONFIG_BT_GATT_DIS`` prefix.
+  It can be configured using Kconfig options with the :option:`CONFIG_BT_GATT_DIS` prefix.
```
Comment (greg-fer) [inline applications/nrf_desktop/README.rst:1314]: @MarekPieta , can you check these options after rebase?

**Review by greg-fer — APPROVED** [review]
Please rebase and squash my edits.

File: applications/nrf_desktop/README.rst (hunk 1 above)
Comment (MarekPieta) [reply applications/nrf_desktop/README.rst:1314]: This is actually used as a prefix here, so I would prefer not to make it an option.

Comment (greg-fer) [reply applications/nrf_desktop/README.rst]: OK, then remove :option:.

**Review by pdunaj — APPROVED** [review]

**Pull request merged**